packages/reactive-element/README.md

```diff
@@ -26,10 +26,12 @@ Full documentation is available at [lit-element.polymer-project.org](https://lit
 ## Overview
 
 ReactiveElement is a low level base class for creating custom elements that
-render content to shadowRoot via a user implemented method. It adds API to help
-manage element properties and attributes. ReactiveElement reacts to changes in
-properties and calls the update method which should be implemented to update
-the rendered state of the element.
+render content to [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM)
```

Comment:
I think it might be clearer if we lead with what ReactiveElement does/provides, instead of rendering and Shadow DOM. Like:

ReactiveElement is a base class for writing web components that react to changes in properties and attributes. ReactiveElement adds reactive properties and a batching, async update lifecycle to the standard web component APIs. Subclasses can respond to changes and update the DOM to reflect the element state.

ReactiveElement doesn't include a DOM template system, but can easily be extended to add one by overriding the `update()` method to call the template library. LitElement is such an extension that adds lit-html templating.

Reply:
Done.